custom_components/toyota/binary_sensor.py

```diff
@@ -266,6 +271,7 @@ class ToyotaOverAllStatusBinarySensor(ToyotaBaseEntity, BinarySensorEntity):
     """Class for the overall warning sensor"""
 
     _attr_device_class = DEVICE_CLASS_PROBLEM
+    _attr_entity_category = ENTITY_CATEGORY_DIAGNOSTIC

```

Comment:
This should not be listed as diagnostic. It is meant for people to easily see if something is not right with car.